src/pu-emmc.c

```diff
@@ -660,6 +662,19 @@ pu_emmc_parse_raw(PuEmmc *emmc,
         g_debug("Parsed raw input: uri=%s md5sum=%s sha256sum=%s",
                 input->uri, input->md5sum, input->sha256sum);
 
+        PedSector min_offset = 1;
+        if (g_str_equal(emmc->disktype->name, "msdos"))
+            min_offset = PARTITION_TABLE_SIZE_MSDOS;
+        else if (g_str_equal(emmc->disktype->name, "gpt"))
+            min_offset = PARTITION_TABLE_SIZE_GPT;
```

Comment:
This section looks identical to the one in `pu_emmc_parse_partitions()`. Please create a small function for better reuse.

Reply:
Created a new function `pu_emmc_get_partition_table_size()`.